app/views/components/_markdown_editor.html.erb

```diff
@@ -15,9 +16,11 @@
           <md-link class="app-c-markdown-editor__toolbar-button" title="Link" aria-label="Link">
             <%= render "components/markdown_editor/link.svg" %>
           </md-link>
-          <md-unordered-list class="app-c-markdown-editor__toolbar-button" title="Bullets" aria-label="Bullets">
-            <%= render "components/markdown_editor/bullets.svg" %>
-          </md-unordered-list>
+          <% if !hide_bullets_button %>
```

Comment:
```suggestion
          <% unless hide_bullets_button %>
```

Reply:
Thanks for the suggestion @DilwoarH. I always forget about this useful `unless` statement, for some reason.